core/remoteviewserver.cpp

```diff
@@ -91,7 +93,6 @@ bool RemoteViewServer::isActive() const
 
 void RemoteViewServer::sendFrame(const RemoteViewFrame &frame)
 {
-    m_clientReady = false;
```

Comment:
The reason was specified in the commit message as "It was possible to trigger a new update request if sourceChanged
was called in between requestUpdateTimeout and sendFrame."
The sourceChanged being directly connected to the QuickOverlay::sceneChanged() signal.
In rare case the signal is emitted in between the 2 quoted calls which trigger a non wanted update request.